fern/pages/docs/getting-started/global-configuration.mdx

```diff
@@ -390,3 +395,18 @@ Be sure the repository is set to **public** visibility.
 <ParamField path="github.branch" type="string" required={true}>
   The branch of the repository you would like the GitHub editor to open a PR to. Default is `main`. 
 </ParamField>
+
+## Landing page configuration
```

Comment:
@elissa-alarmani think it'd be helpful to link out to Cohere's example that's open source and live in production?